src/ktopt_planning_context.cpp

```diff
@@ -248,7 +303,8 @@ void KTOptPlanningContext::setRobotDescription(std::string robot_description)
   auto [plant, scene_graph] = AddMultibodyPlantSceneGraph(builder.get(), 0.0);
 
   // TODO:(kamiradi) Figure out object parsing
-  // auto robot_instance = Parser(plant_, scene_graph_).AddModelsFromString(robot_description_, ".urdf");
+  // auto robot_instance = Parser(plant_,
+  // scene_graph_).AddModelsFromString(robot_description_, ".urdf");
```

Comment:
weird line break, is this from an autoformatter?

Reply:
yup. Ill change it